src/BlogContent/AQUA-Evaluation/Aqua_eligible_testqueries.sql

```diff
@@ -0,0 +1,35 @@
+ select  count(*) from amazon_reviews where product_title SIMILAR TO  '%lap%' or
+product_title SIMILAR TO  '%iph%' or 
```

Comment:
two spaces here... this all looks haphazard, can you clean up formatting please.